CONTRIBUTING.md

```diff
@@ -728,8 +728,8 @@ Think of `*:development:*` scripts as long-running (`--serve`, `--watch`) and mu
 ### Why are `assets` and `baseline-screenshots` protected branches on GitHub?
 
 All of our code is deployed to a single Cloudflare R2 bucket.
-Each branch is [deployed](https://github.com/CrowdStrike/glide-core/blob/main/.github/workflows/on-pull-request-opened-and-synchronize.yml#L126) to a directory based on the branch name.
```

Comment:
These links are already outdated. The links are helpful. But I thought it best to remove them. They'll be out of date again before long.